packages/client/test/client.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { expect } from 'chai';
+import { ec as ECDSA } from 'elliptic';
+import { KeyType, Property, PropertyType } from '@0auth/message';
+import { authPrivacy } from '@0auth/server';
+import { getSignature, StorageType, storeSignature } from '../src';
+import {
+  decryptMessage,
+  encryptMessage,
+  getDecryptedMessageFromLocalStorage,
+  getKeyFromLocalStorage,
+  getMessageFromLocalStorage,
+  storeKeyToLocalStorage,
+  storeMessageToLocalStorage,
+} from '../src/utils';
+
+function mockStorage() {
+  let storage: { [key: string]: string } = {};
+  return {
+    setItem: (key: string, value: string) => (storage[key] = value),
+    removeItem: (key: string) => delete storage[key],
+    getItem: (key: string) => (storage[key] !== undefined ? storage[key] : null),
+    clear: () => (storage = {})
+  };
+}
```

Comment:
Defined for mock testing because there is no localStorage in the node.js execution environment.